src/components/Quill/Quill.tsx

```diff
@@ -56,13 +57,15 @@ export class Quill extends React.Component<IProps, IState> {
     }
 
     return (
-      <this.WebViewComponent
-        source={{ html: this.state.html }}
-        javaScriptEnabled={true}
-        style={this.webViewStyle}
-        onMessage={this.onMessage}
-        ref={this.registerWebView}
-      />
+      <View style={this.props.containerStyle}>
```

Comment:
```suggestion
      <View style={this.props.containerStyle || {}}>
```

Kijk even of `style={undefined}` geen error geeft, want `this.props.containerStyle` kan dus `undefined` zijn als iemand die property niet mee geeft.

Als dat een error geeft is bovenstaande wijziging een oplossing daarvoor